c10/xpu/XPUCachingAllocator.cpp

```diff
@@ -1209,6 +1224,14 @@ class XPUAllocator : public DeviceAllocator {
     assertValidDevice(device);
     device_allocators[device]->resetAccumulatedStats();
   }
+
+  void enablePeerAccess(c10::DeviceIndex dev, c10::DeviceIndex dev_to_access) {
+    assertValidDevice(dev);
+    assertValidDevice(dev_to_access);
+    c10::xpu::get_raw_device(dev).ext_oneapi_enable_peer_access(
```

Comment:
Should we handle the exception if failed? Or, you assume the ext_oneapi_can_access_peer already checked before calling here?